apps/evm/src/clients/api/queries/getIsolatedPools/formatOutput/index.ts

```diff
@@ -50,23 +51,23 @@ const formatToPools = ({
   userVTokenBalancesAll,
   userTokenBalancesAll,
 }: FormatToPoolsInput) => {
-  const pools: Pool[] = poolResults.map(poolResult => {
+  const formattedPools: Pool[] = pools.map(p => {
     const subgraphPool = poolParticipantsCountResult?.pools.find(pool =>
-      areAddressesEqual(pool.id, poolResult.comptroller),
+      areAddressesEqual(pool.id, p.address),
     );
 
-    const assets = poolResult.vTokens.reduce<Asset[]>((acc, vTokenMetaData) => {
+    const assets = p.markets.reduce<Asset[]>((acc, market) => {
       // Remove unlisted tokens
-      if (!vTokenMetaData.isListed) {
+      if (!market.isListed) {
         return acc;
       }
 
       const underlyingTokenAddress =
         // If underlying asset address is the null address, this means the VToken has no underlying
         // token because it is a native token
-        areAddressesEqual(vTokenMetaData.underlyingAssetAddress, NULL_ADDRESS)
+        areAddressesEqual(market.underlyingTokenAddress, NULL_ADDRESS)
           ? NATIVE_TOKEN_ADDRESS
-          : vTokenMetaData.underlyingAssetAddress;
+          : market.underlyingTokenAddress;
```

Comment:
This logic isn't needed anymore since it's present here already: https://github.com/VenusProtocol/venus-protocol-interface/pull/3156/files#diff-c9a8841d55105945315df70e596199b85a70aa24bedaf507f506ae8b69a6217cR20